### PR TITLE
update examples to load prebid and gpt in parallel

### PR DESCRIPTION
**Pull request by protonate**
Also updates to code style.

### CODE REVIEW

**Comment by mkendall07** [comment]
@protonate 
Can you update the example like this fiddle? http://jsfiddle.net/48pLvsqw/3/

Namely, moving `disableInitialLoad` to the top, along with `setTargetingForGPTAsync` and `refresh` into the `initAdserver` function.

**Comment by protonate** [comment]
Updated.

File: integrationExamples/gpt/pbjs_partial_refresh_gpt.html [hunk 1]
```diff
@@ -2,270 +2,292 @@
...
+                    728,
+                    90
+                ],
+                [
+                    970,
+                    90
+                ]
+            ], 'div-gpt-ad-12345678-1').addService(googletag.pubads());
+
+            pbjs.que.push(function () {
```
Comment (mkendall07) [inline integrationExamples/gpt/pbjs_partial_refresh_gpt.html]: @protonate 
this whole function block can be removed now.

**Comment by protonate** [comment]
Ok, updated to remove unnecessary code blocks.

**Comment by protonate** [comment]
@mkendall07 I'm going to hold off on rebasing this until #297 is merged as they both touch the example pages.

**Comment by mkendall07** [comment]
Thanks @protonate

**Pull request merged**